playbooks/init/repos.yml

```diff
@@ -1,8 +1,12 @@
 ---
 - name: Setup yum repositories for all hosts
   hosts: oo_all_hosts
+  become: true
   gather_facts: no
   tasks:
+  - name: subscribe instance to Red Hat Subscription Manager
+    include_role:
+      name: rhel_subscribe
```

Comment:
It looks like including this role without defining the variables it expects will result in a failure. Only include the role when the required variables are defined?